.github/workflows/build.yml

```diff
@@ -17,14 +17,14 @@ jobs:
     strategy:
       matrix:
         scalaVersion: ['2.12.10', '2.13.1']
-        javaTag: ['8u222', '11.0.5', '11.0.2-oraclelinux7', 'graalvm-ce-19.3.0-java11', 'graalvm-ce-19.3.0-java8']
+        javaTag: ['8u242', '11.0.6', '11.0.2-oraclelinux7', 'graalvm-ce-19.3.0-java11', 'graalvm-ce-19.3.0-java8']
         include:
-          - javaTag: '8u222'
+          - javaTag: '8u242'
             dockerContext: 'debian'
-            baseImageTag: '8u222-jdk-stretch'
+            baseImageTag: '8u242-jdk-stretch'
           - javaTag: '11.0.5'
```

Comment:
You should change the version in this line too! `javaTag: '11.0.5'` -> `javaTag: '11.0.6'`